core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -177,22 +177,31 @@ public class TableWriter implements TableWriterAPI, MetadataChangeSPI, Closeable
     private long lastPartitionTimestamp;
     private LifecycleManager lifecycleManager;
     private int lockFd = -1;
-    private long masterRef = 0;
+    private long masterRef = 0L;
```

Comment:
second person to ask. mmmm, maybe I am wrong. I like to add the L at the end of the 0, to me it feels like otherwise it is missing something. I suppose the compiler would do the cast implicitly, I like the L, I would like to keep it.

Reply:
Our L usage is not very consistent. I'm more than fine with using it everywhere or not using it at all. As for this PR, no problem with keeping the literal changes.